docker-compose.yml

```diff
@@ -1,6 +1,6 @@
 web:
-    build: .
-    ports:
-        - "8888:8888"
-    working_dir:
-        - .:/notebook
+  build: ./spark
```

Comment:
That's mainly for my own purposes as I intend to add some additional containers in the forked repo - I had only meant to do a pr for the first changes, but neglected to branch them first. I'll figure out a way to fix it.